static/app/views/settings/project/projectReplays.tsx

```diff
@@ -65,6 +75,11 @@ function ProjectReplaySettings({organization, project, params: {projectId}}: Pro
     },
   ];
 
+  const {getParamValue, setParamValue} = useUrlParams(
+    'replay-settings-tab',
```

Comment:
```suggestion
    'replaySettingsTab',
```
are our url params conventionally camel case?